packages/lzma/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: lzma
+  version: 1.0.0 # Nonesense
+  _cpython_dynlib: true
+source:
+  path: empty
```

Comment:
We really ought to have a workaround for this. Maybe we should skip the `source` section entirely if `_cpython_dynlib` is true.

Reply:
Yes, we need a proper workaround before we unvendor more stdlibs...